src/environment.cpp

```diff
@@ -1322,7 +1324,7 @@ void ServerEnvironment::step(float dtime)
 	/*
 		Manage active block list
 	*/
-	if(m_active_blocks_management_interval.step(dtime, 2.0))
+	if (m_active_blocks_management_interval.step(dtime, m_cache_abm_interval))
 	{
 		ScopeProfiler sp(g_profiler, "SEnv: manage act. block list avg /2s", SPT_AVG);
```

Comment:
that means allocating a std::string every serverstep. Do you really want to do that? How about a generic string such as ```SEnv: manage act. block list avg /int``` instead?